### ISSUE TITLE
Add WCSAxes to affiliated package repo

### ISSUE THREAD

**Issue by astrofrog** [astropy/astropy.github.com#46]
If we go ahead and merge https://github.com/astropy/astropy/pull/3183 then we should add WCSAxes to the list of affiliated packages (and I will move the repo to the astropy organization).

**Comment by sudheesh001** [comment]
@astrofrog It looks like astropy/astropy#3183 has been merged. So this issue could be taken up and patched. Can you review if the following information is correct (and provide information where I missed out), I would be more than happy to send a Pull request.

```
Package Name: WCSAxes
Stable: 
Provisional: 
PyPI name: wcsaxes
Web page: https://github.com/astrofrog/wcsaxes
Code repository: https://github.com/astrofrog/wcsaxes.git
Maintainer: Thomas Robitaille
```

**Comment by astrofrog** [comment]
@sudheesh001 - yes, this looks correct, thanks! Please set stable to Yes and Provisional to Yes.

**Comment by sudheesh001** [comment]
Fixed in commit b05d746 This is good to close @astrofrog

**Comment by astrofrog** [comment]
Thanks @sudheesh001!
